.github/workflows/reusable_go_lint_test.yml

```diff
@@ -41,10 +41,15 @@ on:
         description: 'Run check-mock-gen'
         type: boolean
         default: false
+      run-build:
+        description: 'Run build'
+        type: boolean
+        default: false
```

Comment:
How about setting the default to true? So that this change does not require update for the rest of the projects?

Reply:
It goes against other inputs as if we put as `false`, we will know which jobs will be run on the caller workflow.
Other repos still have the build process running because they use the previous version of the workflow

Reply:
> It goes against other inputs as if we put as false, we will know which jobs will be run on the caller workflow.

Indeed.

>  Other repos still have the build process running because they use the previous version of the workflow

That's a bug, not a feature :smile: 
Overall, we need to track and ideally always bump to latest the pipeline version on every repo - right now we haven't even onboarded all services to GH Actions and we already start seeing diversions.